### PR TITLE
feat(server-sidebar): add anthos entry

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/anthos`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-7384
| License          | BSD 3-Clause

## Description

- Add Anthos entry in the sidebar
- Rename Hosted Private Cloud to VMWare

### CODE REVIEW

**Review by varun257 — APPROVED** [review]

File: packages/manager/modules/server-sidebar/src/sidebar.constants.js [hunk 1]
```diff
@@ -92,6 +92,25 @@ export const MANAGED_BAREMETAL_CONFIG = {
...
+      stateParams: ['productId'],
+      icon: 'ovh-font ovh-font-dedicatedCloud',
+      app: [DEDICATED],
+      namespace: HPC_NAMESPACE,
+    },
+  ],
+  loadOnState: 'app.dedicated-anthos.details',
+  icon: 'ovh-font ovh-font-dedicatedCloud',
+  app: [DEDICATED],
+  regions: ['EU', 'CA'],
```
Comment (radireddy) [inline packages/manager/modules/server-sidebar/src/sidebar.constants.js:110]: please use feature flipping

**Review by radireddy — CHANGES_REQUESTED** [review]

File: packages/manager/modules/server-sidebar/src/sidebar.constants.js (hunk 1 above)
Comment (ganeshkumar1989) [reply packages/manager/modules/server-sidebar/src/sidebar.constants.js]: This has been done :)

**Review by radireddy — APPROVED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Review by antleblanc — CHANGES_REQUESTED** [review]
Tests failed. Looks like the issue is located into the targeted branch `feat/anthos`.
A rebase will be required.

**Comment by ganeshkumar1989** [comment]
> Tests failed. Looks like the issue is located into the targeted branch `feat/anthos`.
> A rebase will be required.

This is done. Thanks!

**Review by marie-j — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**